### PR TITLE
Fixed #25465 -- Returned line breaks addition to plain readonly fields in admin

### PR DESCRIPTION
**Pull request by sasha0**

### CODE REVIEW

**Comment by timgraham** [comment]
As you can see, this broke some tests. I think the correct resolution would be to apply that filter somewhere in Python code. Also, we need a regression test for this fix.

**Comment by sasha0** [comment]
@timgraham ok done.

**Comment by sasha0** [comment]
@timgraham ok done.

File: tests/admin_views/tests.py [hunk 1]
```diff
@@ -4662,6 +4662,17 @@ def test_readonly_get(self):
         response = self.client.get(reverse('admin:admin_views_post_change', args=(p.pk,)))
         self.assertContains(response, "%d amount of cool" % p.pk)
 
+    @ignore_warnings(category=RemovedInDjango20Warning)  # for allow_tags deprecation
+    def test_readonly_text_field(self):
+        p = Post.objects.create(title="Readonly test", content="test",
+                                readonly_content=u'test\r\n\r\ntest\r\n\r\ntest\r\n\r\ntest')
```
Comment (timgraham) [inline tests/admin_views/tests.py]: no u'' prefixes -- we use `from __future__ import unicode_literals`

File: tests/admin_views/tests.py [hunk 2]
```diff
@@ -4662,6 +4662,17 @@ def test_readonly_get(self):
         response = self.client.get(reverse('admin:admin_views_post_change', args=(p.pk,)))
         self.assertContains(response, "%d amount of cool" % p.pk)
 
+    @ignore_warnings(category=RemovedInDjango20Warning)  # for allow_tags deprecation
+    def test_readonly_text_field(self):
+        p = Post.objects.create(title="Readonly test", content="test",
```
Comment (timgraham) [inline tests/admin_views/tests.py]: Please use 4 space hanging indent style as seen elsewhere in this file such as `cls.superuser = ...`

File: tests/admin_views/test [hunk 3]
```diff
@@ -0,0 +1 @@
...n    \n  </ul>\n\n\n<form enctype="multipart/form-data" action="" method="post" id="post_form" novalidate><input type=\'hidden\' name=\'csrfmiddlewaretoken\' value=\'vm6UKOvf2bLmFhzT1BpDr65o32goUN86\' />\n<div>\n\n\n\n\n\n\n\n  \n\n\n\n\n\n\n\n\n\n\n\n\n<div class="submit-row">\n<input type="submit" value="Save" class="default" name="_save" />\n\n    \n    <p class="deletelink-box"><a href="/test_admin/admin/admin_views/post/1/delete/" class="deletelink">Delete</a></p>\n\n\n<input type="submit" value="Save and add another" name="_addanother" />\n<input type="submit" value="Save and continue editing" name="_continue" />\n</div>\n\n\n\n    <script type="text/javascript"\n            id="django-admin-form-add-constants"\n            src="/static/admin/js/change_form.js"\n            >\n    </script>\n\n\n\n\n<script type="text/javascript"\n        id="django-admin-prepopulated-fields-constants"\n        src="/static/admin/js/prepopulate_init.js"\n        data-prepopulated-fields="[]">\n</script>\n\n\n</div>\n</form></div>\n\n        \n        <br class="clear" />\n    </div>\n    <!-- END Content -->\n\n    <div id="footer"></div>\n</div>\n<!-- END Container -->\n\n</body>\n</html>\n'
```
Comment (timgraham) [inline tests/admin_views/test]: Inadvertently committed this file?

Comment (sasha0) [inline tests/admin_views/test]: Yep.

**Comment by timgraham** [comment]
If this is a regression in 1.8, it qualifies for a backport, in which case, please add release notes for 1.9.1 and 1.8.8.

**Comment by sasha0** [comment]
Ok done.

**Comment by timgraham** [comment]
merged in 69208a5a1c55d42ca0eaffa900be643d9f801089, thanks!

**Pull request closed without merging**